src/Amadeus/Client/RequestOptions/Service/StandaloneCatalogue/Passenger.php~

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
What is the point of this file? It should be removed.

Reply:
I meant the file with the tilde at the end (`.php~`)

Reply:
done